From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Wed, 2 Sep 2020 20:21:26 -0300
Subject: [PATCH 11/17] [FIO extras] fastboot: don't enable console mux by
 default

This is required for enabling console/uart output at uuu, but this
shouldn't be set by default as it doesn't work properly with every
target yet (e.g. imx7ulp).

Users can still set both options manually if they want such
functionality.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
(cherry picked from commit 8c2581a62d9d783e388ff8fe4c5bf41ea1a25245)
---
 drivers/fastboot/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/fastboot/Kconfig b/drivers/fastboot/Kconfig
index 67cacb09721..f81313a29b8 100644
--- a/drivers/fastboot/Kconfig
+++ b/drivers/fastboot/Kconfig
@@ -108,8 +108,6 @@ config FASTBOOT_UUU_SUPPORT
 	select CMD_GPT
 	select RANDOM_UUID
 	select CMD_GPT_RENAME
-	select CONSOLE_MUX
-	select SYS_STDIO_DEREGISTER
 	help
 	  This extends the fastboot protocol with the "UCmd" and "ACmd"
 	  commands, which are used by NXP's "universal update utility" (UUU).
